aidial_sdk/application.py

```diff
@@ -42,6 +44,7 @@ def filter(self, record: LogRecord):
 
 class DIALApp(FastAPI):
     chat_completion_impls: Dict[str, ChatCompletion] = {}
+    embeddings_impls: Dict[str, Embeddings] = {}
```

Comment:
Why do we move `Embeddings` inside of distinct class, but `tokenize`, for instance are included inside of `ChatCompletion`?

I think we either should

1. Rename `ChatCompletion` in something like `ModelDeployment` and include embeddings inside of this ModelDeployment
2. Create an issue, to move tokenize and truncate prompt into distinct classes/functions, and `ChatCompletion` class does only chat completion

Reply:
`tokenize` and `truncate_prompt` are only relevant for chat/completion models. It doesn't make much sense to implement them independently of chat/completion main endpoint.

```
/chat/completions
   - /tokenize (optional)
   - /truncate_prompt (optional)
   - /rate (optional)
/embeddings
```

Reply:
`/rate` is for like/dislike feedback from a user. It's not relevant for embedding models.

I agree that `/tokenize` could be useful for embedding models.